subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -2449,11 +2449,7 @@ static inline int event_conn_upd_prep(struct ll_conn *conn, uint16_t lazy,
 				     ticks_at_expire, ticks_win_offset,
 				     HAL_TICKER_US_TO_TICKS(periodic_us),
 				     HAL_TICKER_REMAINDER(periodic_us),
-#if defined(CONFIG_BT_CTLR_CONN_META)
-				     TICKER_LAZY_MUST_EXPIRE,
-#else
-				     TICKER_NULL_LAZY,
-#endif /* CONFIG_BT_CTLR_CONN_META */
+				     TICKER_LAZY_MUST_EXPIRE_KEEP,
```

Comment:
Not sure i understand this one.

Reply:
The idea is that your ticker node "must-expire" state lasts for the duration of the connection. At connection update, ticker is stopped and started again. If it were not for the addition of a "KEEP" flag, re-starting the ticker would either enable or disable "must-expire", regardless of what you requested in the ticker_update at an earlier point. The "KEEP" flag tells the ticker_start to not change the state.

Reply:
@thoh-ot, thanks for pointing out the missing COMPATIBILIY_MODE flag in ull_conn. Now updated.